j/jmxtrans/jmxtrans_ubi8_3.sh

```diff
@@ -14,18 +14,16 @@
 #             contact "Maintainer" of this script.
 #
 # ----------------------------------------------------------------------------
-
 #!/bin/bash
 
 #Variables
 PACKAGE_URL=https://github.com/jmxtrans/jmxtrans.git
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
 PACKAGE_VERSION=jmxtrans-parent-272
```

Comment:
Replace this with 
`PACKAGE_VERSION="${1:-jmxtrans-parent-272}"`

Reply:
Okay.